scripts/svm/simpleFill.ts

```diff
@@ -49,7 +50,7 @@ async function fillV3Relay(): Promise<void> {
   const inputAmount = new BN(resolvedArgv.inputAmount);
   const outputAmount = new BN(resolvedArgv.outputAmount);
   const originChainId = new BN(resolvedArgv.originChainId);
-  const depositId = resolvedArgv.depositId;
+  const depositId = intToU8Array32(resolvedArgv.depositId);
```

Comment:
this might not work with deterministic ids because of JS number size limitations. maybe better let `resolvedArgv.depositId` be string and parse it as BigNum->U8Array32 if it starts with 0x?

Reply:
ye, this is a good point. I've updated accordingly and updated `intToU8Array32` to support BN.